lib/ecto/migration.ex

```diff
@@ -474,6 +474,17 @@ defmodule Ecto.Migration do
   end
 
   @doc """
+  Renames a column outside of the `alter` statement. Note that unlike `rename/3` it is not necessary
```

Comment:
The first line of docs should always be a short summary. Just a note, I have already fixed this locally. :)